settings.gradle.kts

```diff
@@ -138,37 +139,37 @@ dependencyResolutionManagement {
             library("micrometer-jetty", "org.eclipse.edc", "jetty-micrometer").versionRef("edc")
             library("monitor-jdklogger", "org.eclipse.edc", "monitor-jdk-logger").versionRef("edc")
             library(
-                "transfer.dynamicreceiver",
-                "org.eclipse.edc",
-                "transfer-pull-http-dynamic-receiver"
+                    "transfer.dynamicreceiver",
```

Comment:
The indention looks weird for me. 8 spaces? Is this correct?

Reply:
nope, thx